skymp5-server/cpp/lib_espm/espm.cpp

```diff
@@ -516,6 +494,24 @@ struct espm::Browser::Impl
   std::vector<std::unique_ptr<GroupDataInternal>> grDataHolder;
 
   CompressedFieldsCache dummyCache;
+
+  // may return null
+  const GroupStack* GetParentGroupsOptional(const RecordHeader* rec) const {
```

Comment:
I would prefer returning a reference to empty GroupStack if possible. If not, that's ok.

Reply:
That's technically possible, we can return a reference to a static variable. Howerver it will mean that existing return value doesn't mean it's present (we change 'non-empty or null' to 'non-empty or empty'). I don't like this option, but I guess it will be more consistent? If so, I'll change.

Reply:
ok, let's leave it